### PR TITLE
style: Make index page look better (#90)

### PR DESCRIPTION
**Pull request by mrmir**
Fixes issue: #90 
Deployment: https://mrmir2.herokuapp.com/

### CODE REVIEW

File: views/index.hbs [hunk 1]
```diff
@@ -22,35 +22,38 @@
...
+                          <span class="sty">TAKE ME THERE</span>
+                      </div>
+                  </div>
+              </a>
+          </div>
+
     </div>
     <div class="row">
         <div class="sixteen wide column">
+            <br>
```
Comment (sumedh123) [inline views/index.hbs]: use semantic's class here. ```hidden divider```

Comment (sumedh123) [reply views/index.hbs]: deployment?

**Comment by mrmir** [comment]
@sumedh123 Please review!

**Comment by sumedh123** [comment]
@mrmir Please change the commit message. also rebase once after fetching all changes.

**Pull request merged**